### PR TITLE
conf.py: use print() function

### PR DESCRIPTION
**Pull request by ktdreyer**
To be compatible with Python 3.

Related: https://bugzilla.redhat.com/1328199

### CODE REVIEW

**Comment by ktdreyer** [comment]
I have tested this on CentOS 5 (Python 2.4.3) and it works fine. `PyConfigParser(debug=True)` causes the parser to print things to stdout.

File: kobo/conf.py [hunk 1]
```diff
@@ -160,7 +160,7 @@ def _get_token(self, skip_newline=True):
         self._tok_name = token.tok_name.get(self._tok_number, None)
 
         if self._debug:
-            print "%2s %16s %s" % (self._tok_number, self._tok_name, self._tok_value.strip())
+            print("%2s %16s %s" % (self._tok_number, self._tok_name, self._tok_value.strip()))
```
Comment (dmach) [inline kobo/conf.py]: I'm fine with that if you add:
from __future__ import print_function
to preserve python2.6 functionality

Comment (ktdreyer) [inline kobo/conf.py]: Thanks, I've amended the commit and re-pushed.

Comment (ktdreyer) [inline kobo/conf.py]: this needs to be the first import, so I've amended again and re-pushed.

**Pull request merged**